packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -1057,3 +1064,14 @@ export enum EbsDeviceVolumeType {
    */
   SC1 = 'sc1',
 }
+
+/**
+ * Stringify a number directly or lazily if it's a Token
+ */
+function stringifyNumber(x: number) {
```

Comment:
I see this problem in quite a few places now where CloudFormation defines an attribute's data type as a string when it should be clearly number. The [AllocatedStorage property](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html) in `AWS::RDS::DBInstance` is another example. 

Do we need this method somewhere that other constructs can use them as well? If so, let's create an issue to track this?

Reply:
You are right.

Reply:
https://github.com/aws/aws-cdk/issues/3769

Reply:
LGTM.